python/lsst/afw/geom/coordinates/coordinates.cc

```diff
@@ -0,0 +1,509 @@
+        self = floor(self / static_cast<double>(other));
+        return self;
+    });
+
+    clsI.def("__idiv__", [](Extent<int, N> &self, int other) -> Extent<int, N> & {
+        self = floor(self / static_cast<double>(other));
+        return self;
+    });
+    clsD.def("__idiv__", [](Extent<double, N> &self, double other) -> Extent<double, N> & {
+        self /= other;
+        return self;
+    });
+
+    clsI.def("__itruediv__", [](Extent<int, N> &self, double other) {
+        PyErr_SetString(PyExc_TypeError, "In-place true division not supported for Extent<int,N>.");
+        throw py::error_already_set();
+    });
+    clsD.def("__itruediv__", [](Extent<double, N> &self, double other) -> Extent<double, N> & {
+        self /= other;
+        return self;
+    });
+
+    clsI.def("__iadd__", [](Extent<int, N> &self, Extent<int, N> const &other) -> Extent<int, N> & {
+        self += other;
+        return self;
+    });
+    clsD.def("__iadd__", [](Extent<double, N> &self, Extent<double, N> const &other) -> Extent<double, N> & {
+        self += other;
+        return self;
+    });
+    clsD.def("__iadd__", [](Extent<double, N> &self, Extent<int, N> const &other) -> Extent<double, N> & {
+        self += other;
+        return self;
+    });
+
+    clsI.def("__isub__", [](Extent<int, N> &self, Extent<int, N> const &other) -> Extent<int, N> & {
+        self -= other;
+        return self;
+    });
+    clsD.def("__isub__", [](Extent<double, N> &self, Extent<double, N> const &other) -> Extent<double, N> & {
+        self -= other;
+        return self;
+    });
+    clsD.def("__isub__", [](Extent<double, N> &self, Extent<int, N> const &other) -> Extent<double, N> & {
+        self -= other;
+        return self;
+    });
+
+    clsI.def("__imul__", [](Extent<int, N> &self, int other) -> Extent<int, N> & {
+        self *= other;
+        return self;
+    });
+    clsD.def("__imul__", [](Extent<double, N> &self, int other) -> Extent<double, N> & {
+        self *= other;
+        return self;
+    });
+    clsD.def("__imul__", [](Extent<double, N> &self, double other) -> Extent<double, N> & {
+        self *= other;
+        return self;
+    });
+
+    // Operator-like free functions
+    mod.def("truncate", truncate<N>);
+    mod.def("floor", floor<N>);
+    mod.def("ceil", ceil<N>);
+    // And method versions, since Python doens't have ADL
+    clsD.def("truncate", truncate<N>);
+    clsD.def("floor", floor<N>);
+    clsD.def("ceil", ceil<N>);
+}
+
+
+template <typename T, int N>
+void declarePointBase(py::module &mod, std::string const & suffix) {
+    declareCoordinateBase<Point<T,N>,T,N>(mod, "Point" + suffix);
+    static std::string const name = "PointBase" + suffix;
+    PyPointBase<T,N> cls(mod, name.c_str());
+
+    /* Operators */
+    cls.def("eq", [](PointBase<T, N> const & self, Point<T, N> const & other) { return self.eq(other); });
```

Comment:
Again...a brief code comment explaining the decision to not spell these `__eq__`, etc. would be appreciated.